src/deadline/blender_adaptor/BlenderClient/render_handlers/default_blender_handler.py

```diff
@@ -96,31 +87,44 @@ def start_render(self, data: dict) -> None:
             data (dict): The data given from the Adaptor. Keys expected: ['frame', 'camera']
 
         Raises:
-            RuntimeError: If no camera was specified and no renderable camera was found
+            RuntimeError: If no camera was specified, no renderable camera was found, or required settings weren't provided
         """
 
         frame = data.get("frame")
         if frame is None:
             raise RuntimeError("BlenderClient: start_render called without a frame number.")
 
-        bpy.context.scene.frame_start = frame
-        bpy.context.scene.frame_end = frame
+        bpy.context.scene.frame_set(frame)
+        _logger.debug(f"Set frame {frame} to render")
 
         # The camera fetched here is only for logging purposes.
         # The actual camera to render is set in the set_camera function,
         # and should have already been called as an Action.
         camera = self._ensure_camera(data)
 
-        # Add layer and camera info to the output file path.
-        bpy.context.scene.render.filepath = (
-            f"{self.output_dir}/{self.view_layer_name}_{camera}_{self.output_file_name}"
-        )
+        # Add submitter settings to the output filepath.
+        # This includes output directory, view layer, camera, and frame.
+        try:
+            bpy.context.scene.render.filepath = (
+                f"{self.output_dir}/{self.view_layer_name}_{camera}_{self.output_file_name}"
+            )
+        except:
```

Comment:
nit: a bare except will catch exceptions like `KeyboardInterrupt` or `SystemExit`. In this case it doesn't really matter because we re-raise, but as a best practice we will want at the minimum 
```suggestion
        except Exception:
```